ts/src/huobi.ts

```diff
@@ -940,43 +940,132 @@ export default class huobi extends Exchange {
                     'BTC': 'BTC',
                     'USDT': 'TRC20',
                 },
+                'networksAreTitlesInsteadOfIds': true,
```

Comment:
@ttodua I'm not seeing any usage for this option

Reply:
in other PRs where I have unified things in base, this would be used (because the networks mapping in huobi are not real network mappings), and it's not only used programmatically, but also tells users that 'network-ids' are not real in the "networks" mapping.

Reply:
I can remove this line from this specific PR, but will need to add it again later